distribution/zip/jballerina-tools/LICENSE

```diff
@@ -1,284 +1,282 @@
-
-This product is licensed by WSO2 Inc. under Apache License 2.0. The license
-can be downloaded from the following locations:
-	http://www.apache.org/licenses/LICENSE-2.0.html
-	http://www.apache.org/licenses/LICENSE-2.0.txt

```

Comment:
The license should change with stdlib migration. But I'm not quite sure whether it is the best option to commit the changed license.

Reply:
Noted. Removed the change in 49bd466